pandas/tests/io/excel/test_readers.py

```diff
@@ -624,7 +624,6 @@ def test_read_from_http_url(self, read_ext):
         local_table = pd.read_excel("test1" + read_ext)
         tm.assert_frame_equal(url_table, local_table)
 
-    @td.skip_if_not_us_locale
```

Comment:
Does anyone know what the intention was in this skip? I'm in the UK and wanted to make sure I hadn't broken these tests. I couldn't think of any reason why this needed to be in, but have removed it in a separate commit which I have no problem dropping out of this PR if people would prefer.

Reply:
This was added in https://github.com/pandas-dev/pandas/pull/21814. But no idea why (the test doesn't seem to have anything locale specific (like eg date parsing)). But we can see what CI says.
cc @TomAugspurger